test/integration/features/basic.feature

```diff
@@ -1,13 +1,13 @@
 Feature: Basic test
     Checks whether CRC top-level commands behave correctly.
 
-    @darwin @linux @windows
```

Comment:
why change the order, it doesn't make sense and also now this change part of commit message.

Reply:
It was introduce by mistake. I'll change it back